jsonschema-core.xml

```diff
@@ -2095,6 +2104,447 @@
+                            </t>
+                            <t>
+                                Nodes that have a single child are replaced by the child.
+                            </t>
+                        </list>
+                        Branch nodes do not require an error message or an annotation.
+                    </t>
+                    <figure>
+                        <artwork>
+<![CDATA[
+{
+  "valid": false,
+  "keywordLocation": "#",
+  "instanceLocation": "#",
+  "errors": [
+    {
+      "valid": false,
+      "keywordLocation": "#/items/$ref",
+      "absoluteKeywordLocation":
+        "http://example.com/polygon#/definitions/point",
+      "instanceLocation": "#/1",
+      "errors": [
+        {
+          "valid": false,
+          "keywordLocation": "#/items/$ref/required",
+          "absoluteKeywordLocation":
+            "http://example.com/polygon#/definitions/point/required",
+          "instanceLocation": "#/1",
+          "error": "Required property 'y' not found."
+        },
+        {
+          "valid": false,
+          "keywordLocation": "#/items/$ref/additionalProperties",
+          "absoluteKeywordLocation":
+            "http://example.com/polygon#/definitions/point/additionalProperties",
+          "instanceLocation": "#/1/z",
+          "error": "Additional property 'z' found but was invalid."
+        }
+      ]
+    },
+    {
+      "valid": false,
+      "keywordLocation": "#/minItems",
+      "instanceLocation": "#",
+      "error": "Expected at least 3 items but found 2"
+    }
+  ]
+}
+]]>
+                        </artwork>
+                    </figure>
+                </section>
+
+                <section title="Verbose">
+                    <t>
+                        The **Verbose** structure is a fully realized hierarchy that exactly matches
+                        that of the schema.  This structure has applications in form generation and
+                        validation where the error's location is important.
+                    </t>
+                    <t>
+                        The primary difference between this and the **Detailed** structure is that
+                        *all* results are returned.  This includes sub-schema validation results that
+                        would otherwise be removed (e.g. annotations for failed validations,
+                        successful validations inside a `not` keyword, etc.).  Because of this, it
+                        is RECOMMENDED that each node also carry a `valid` property to indicate the
+                        validation result for that node.
+                    </t>
+                    <t>
+                        Because this output structure can be quite large, a smaller example is given
+                        here for brevity.  The full output structure of the example above can be found
+                        [here](standardized-output-verbose.json)
```

Comment:
@handrews how do I do external links in this XML file?  Probably not with Markdown as I have here.

Reply:
@gregsdennis I think you want `<eref>`.  The full XML RFC reference is https://tools.ietf.org/html/rfc7749 (yes, that RFC is marked obsolete, but it's the right one- the xml2rfc tool still doesn't support the new RFC for xml2rfc, ironically).